convokit/storage/convoKitIndex.py

```diff
@@ -0,0 +1,275 @@
+from collections import defaultdict
+from typing import Optional, Dict, List
+import json
+
+
+class ConvoKitIndex:
+    def __init__(self,
+                 storage,
+                 utterances_index: Optional[Dict[str, List[str]]] = None,
+                 speakers_index: Optional[Dict[str, List[str]]] = None,
+                 conversations_index: Optional[Dict[str, List[str]]] = None,
+                 overall_index: Optional[Dict[str, List[str]]] = None,
+                 vectors: Optional[List[str]] = None,
+                 version: Optional[int] = 0):
+
+        self.storage = storage
+        self.fields = defaultdict(dict)
```

Comment:
This is configured as `defaultdict(dict)`, which indicates that the values held in `self.fields` should all be dictionaries. However, this is not true since `version` and `vectors` are stored here. Could we just use a regular dict?

Reply:
Yeah I get the intended functionality -- it's just non-standard code design.

Reply:
Would u recommend just using {} instead then? would we need to handle errors with try except?

Reply:
Yeah a regular dict should suffice. Dict's `get()` is already safe, you can just define a default value if the key is not found, i.e. `dictionary.get('key', dict())` if you want to replicate the `defaultdict(dict)` functionality.